pycbc/events/cuts.py

```diff
@@ -388,7 +388,4 @@ def apply_template_cuts(bank, template_cut_dict, template_ids=None,
             raise ValueError("Cut parameter " + parameter + " not recognised."
                              " This shouldn't happen with input sanitisation")
 
-    logging.info("%d out of %d templates kept after applying template cuts",
```

Comment:
This logging is applied once in the offline codes (pycbc_sngls_findtrigs and pycbc_coinc_findtrigs), but many times (once per file) in pycbc_live_single_trigger_fits, and spams the logging.

I have moved the logging to outside the function for this to be clearer